core/src/main/java/dk/alexandra/fresco/framework/util/DrngImpl.java

```diff
@@ -48,6 +51,19 @@ public BigInteger nextBigInteger(BigInteger limit) {
     return result.compareTo(limit) < 0 ? result : nextBigInteger(limit);
   }
 
+  @Override
+  public boolean nextBit() {
+    if (bitsLeft == 0) {
+      drbg.nextBytes(randomBytes);
+      bitsLeft = RANDOMBUFFER_SIZE * Byte.BYTES * 8;
+    }
+    int index = RANDOMBUFFER_SIZE * Byte.BYTES * 8 - bitsLeft;
+    byte currentByte = randomBytes[index / (Byte.BYTES * 8)];
+    byte currentBit = (byte) (currentByte >> (index % (Byte.BYTES * 8)));
+    bitsLeft--;
+    return currentBit == 0x00 ? false : true;
```

Comment:
Simplifies to `currentBit != 0x00;`. I'm assuming `currentBit == 0x01` should also work.

Reply:
I would prefer to keep as it is now because of increased readability